CallKitDemo/SpeakerboxCall.swift

```diff
@@ -203,6 +203,10 @@ extension SpeakerboxCall: OTSessionDelegate {
     
     func session(_ session: OTSession, didFailWithError error: OTError) {
         print(#function, error)
+        
+        hasConnected = false
+        startCallCompletion?(false)
```

Comment:
How about naming it `canStartCall?(false)` and `canAnswerCall?(false)` ?

Reply:
Updated!